src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Monaco.Edit.cshtml

```diff
@@ -10,75 +10,12 @@
 
 <div class="form-group">
     <label asp-for="Html">@Model.PartFieldDefinition.DisplayName()</label>
-    <div id="@Html.IdFor(x => x.Html)_editor" asp-for="Text" style="min-height: 400px;" class="form-control" dir="@culture.GetLanguageDirection()"></div>
+    <div class="form-control" dir="@culture.GetLanguageDirection()" data-monaco-oc="true" data-linked-id="@Html.IdFor(x => x.Html)" data-settings='@Html.Raw(monacoSettings.Options)' data-shortcodes="true"></div>
```

Comment:
@sebastienros do you have any tips on how to properly encode the `data-settings` here ? I know that the second someone includes a `'` in the monaco options in the admin, the data-settings won't work.

Reply:
Using `IJsonHelper.Serialize(object).ToString()` correctly encodes json as html entities, we should use it more often, it essentially seems to cover everything (and uses the NewtonSoft serializer when that's configured - which we do, so all our attributes work as well)

Be great to revisit this pr when you have time @jptissot